changelog.md

```diff
@@ -537,7 +544,7 @@ Below is a list of any know issues affecting the [recommended minimum version](#
 
 | Issue | Impact | Mitigation | Tracking link |
 | --- | --- | --- | --- |
-| When using Direct mode with an account with multiple write locations, the SDK might not detect when a region is added to the account. The background process that [refreshes the account information](/azure/cosmos-db/troubleshoot-sdk-availability#adding-a-region-to-an-account) fails to start. |If a new region is added to the account which is part of the PreferredLocations on a higher order than the current region, the SDK won't detect the new available region. |Upgrade to 2.17.0. |https://github.com/Azure/azure-cosmos-dotnet-v2/issues/852 |
+| When using Direct mode with an account with multiple write locations, the SDK might not detect when a region is added to the account. The background process that [refreshes the account information](/azure/cosmos-db/troubleshoot-sdk-availability#adding-a-region-to-an-account) fails to start. |If a new region is added to the account which is part of the PreferredLocations on a higher order than the current region, the SDK won't detect the new available region. |Upgrade to 2.18.0. |https://github.com/Azure/azure-cosmos-dotnet-v2/issues/852 |
```

Comment:
This needs to be removed. This table is for known issues on the recommended version. If we bump to 2.18 the recommended version, then this problem is no more.